nucypher/utilities/seednodes.py

```diff
@@ -16,37 +16,28 @@
 
```

Comment:
Good relocation 👍

Reply:
I mean - maintaining executable python code in string format is obviously less maintainable: it is not parsed by linters and is overtly implicit.